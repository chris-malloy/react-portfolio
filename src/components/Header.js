import React from 'react';
import '../stylesheets/Header.css';

const Header = () => {
  return (
    <div className="center blue"> 
      <img src="profile-picture-headshot.jpeg" alt="profile" />
      <h5>Imagination meets Technicality</h5>
    </div>
  )
}

export default Header;