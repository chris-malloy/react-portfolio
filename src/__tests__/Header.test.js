import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Header from '../components/Header';

configure({ adapter: new Adapter() });

describe('Header', () => {
  const header = shallow(<Header />);
  
  it('renders correctly', () => {
    expect(header).toMatchSnapshot();
  })
})