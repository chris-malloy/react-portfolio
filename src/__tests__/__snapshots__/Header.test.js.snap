// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="profile"
          src="profile-picture-headshot.jpeg"
/>,
        <h5>
          Imagination meets Technicality
</h5>,
      ],
      "className": "center blue",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "profile",
          "src": "profile-picture-headshot.jpeg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Imagination meets Technicality",
        },
        "ref": null,
        "rendered": "Imagination meets Technicality",
        "type": "h5",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="profile"
            src="profile-picture-headshot.jpeg"
/>,
          <h5>
            Imagination meets Technicality
</h5>,
        ],
        "className": "center blue",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "profile",
            "src": "profile-picture-headshot.jpeg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Imagination meets Technicality",
          },
          "ref": null,
          "rendered": "Imagination meets Technicality",
          "type": "h5",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
